blocks/shared-styles/scss/_header-nav.scss

```diff
@@ -225,13 +225,6 @@ body.nav-open {
     &.nav-logo, & > .nav-logo {
       justify-content: center;
 
-      img {
-        height: auto;
-        max-height: 40px;
-        max-width: 240px;
-        width: auto;
-      }
-
```

Comment:
I move these lines of this style to header-nav-chain-block. The purpose is logo height will apply animation when user scrolling.